File: src/main/java/com/github/iamniklas/liocore/led/Area.java
package com.github.iamniklas.liocore.led;

public class Area {
    int startIndex;
    int endIndex;

    public Area(int _start, int _end) {
        startIndex = _start;
        endIndex = _end;
    }

    public Area(Area _area) {
        startIndex = _area.startIndex;
        endIndex = _area.endIndex;
    }

    void redefineArea(int _start, int _end) {
        startIndex = _start;
        endIndex = _end;
    }
}

File: src/main/java/com/github/iamniklas/liocorekotlin/led/Area.kt
package com.github.iamniklas.liocorekotlin.led

class Area {
    var startIndex: Int
    var endIndex: Int

    constructor(_start: Int, _end: Int) {
        startIndex = _start
        endIndex = _end
    }

    constructor(_area: com.github.iamniklas.liocorekotlin.led.Area) {
        startIndex = _area.startIndex
        endIndex = _area.endIndex
    }

    fun redefineArea(_start: Int, _end: Int) {
        startIndex = _start
        endIndex = _end
    }
}